Dockerfile

```diff
@@ -58,15 +60,23 @@ RUN set -x && \
         zlibc \
 		# for emojis
 		librsvg2-bin \
+    # add en_US Locale including UTF-8 support
+    && rm -rf /var/lib/apt/lists/* \
+    && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8 \
     # clean up
     && apt-get clean && \
     rm -rf /var/lib/apt/lists/* /etc/apt/apt.conf.d/01proxy
 
+#
+# Set Locale for UTF-8 support
+#
+ENV LANG en_US.UTF-8
+
 #
 # SSH pre-config / useful for Gitlab CI
 #
 RUN mkdir -p ~/.ssh && \
-    echo -e "Host *\n\tStrictHostKeyChecking no\n\n" > ~/.ssh/config
+    echo "Host *\n\tStrictHostKeyChecking no\n\n" > ~/.ssh/config
```

Comment:
Can you explain why ?

Reply:
Hi,

certainly: when building with the `-e` argument, my docker ("version 18.09.2, build 6247962") adds the string `-e` itself into the "config" file. Therefore I removed it.

This is how it looks on opening the generated ssh config file:

```shell
root@3395aea231bc:/pandoc# cat /root/.ssh/config
-e Host *
	StrictHostKeyChecking no
```

Reply:
I've been able to reproduce. it's most probably due to a setup inside the `debian:stretch-slim` image.

Thanks for reporting this, I'll probably extract this from this PR and create another one because this bug has little to do with the new feature

Reply:
alright, I agree!